src/lambdas/breeds-get.test.ts

```diff
@@ -0,0 +1,44 @@
+import { getBreeds } from './breeds-get'
+import { input1, assert1 } from './mock-events/mock-values'
+import * as req from '../services/http-req/dogBreedRequest'
+
+import { Status } from './types'
+
+jest
+  .spyOn(req, 'getDogBreeds')
+  .mockImplementation((): any => ({ status: Status.success, message: input1 }))
+
+describe('test successful API call', () => {
+  it('should return 200 for a successful call', async () => {
+    const response = await getBreeds()
+    expect(response.statusCode).toBe(200)
+  })
+
+  it('should have a correctly formated body array', async () => {
+    const response: any = await getBreeds()
+    expect(response.body.length).toEqual(assert1.length)
+    expect(response.body.join(' ')).toEqual(assert1.join(' '))
```

Comment:
I'm a little curious about the `join(' ')` on both the body and the mock value. Without the join wouldn't these be equal as well?

Reply:
LOL, I have the habit of doing this when comparing arrays. Completely not needed here so it has been removed.